.github/workflows/macos_unit_tests.yml

```diff
@@ -11,9 +11,9 @@ jobs:
       with:
         submodules: recursive
     - name: Install dependencies
-      run: brew install lmdb automake openssl@1.1 pcre
+      run: brew install lmdb automake openssl pcre
```

Comment:
removing the `@1.1` has the effect in github of a no-op. openssl@3 is already installed in their image I guess

``` 
Warning: openssl@3 3.0.8 is already installed and up-to-date.
To reinstall 3.0.8, run:
  brew reinstall openssl@3
```